Fix checks timeout definition in README.md (#213)
canonical/pebble: README.md

        period: <duration>

        # (Optional) If this time elapses before a single check operation has

>         # finished, it is cancelled and considered an error. Must not be more

Reviewer: Let's be direct instead of using the negative:

```suggestion
        # finished, it is cancelled and considered an error. Must be less
```

Reviewer: Additionally the code / error message uses the direct version: https://github.com/canonical/pebble/blob/0c8bb88da360012178b23329cd4a0b395596c467/internal/plan/plan.go#L666
